Kickstarter-iOS/Views/Controllers/SelectCurrencyViewController.swift

```diff
@@ -0,0 +1,79 @@
+import Library
+import Prelude
+import Prelude_UIKit
+import UIKit
+
+final class SelectCurrencyViewController: UIViewController, MessageBannerViewControllerPresenting {
```

Comment:
Do we need to use `UIViewController` as oppose to `UITableViewController`? It would save us couple lines (23, 24 in `viewDidLoad`)...but I'm unsure if it will work properly with the message banner.